package/doc/sphinx/source/documentation_pages/guesser_modules.rst

```diff
@@ -0,0 +1,61 @@
+.. Contains the formatted docstrings from the guesser modules located in 'mdanalysis/package/MDAnalysis/guesser'
+
+**************************
+guesser modules
+**************************
+This module contains the context-aware guessers, which are used by the :ref:`guess_TopologyAttributes <guess_TopologyAttributes>` API. Context-aware guessers' main purpose
+is to be tailored guesser classes that target specific file format or force field (eg. PDB file format, or Martini forcefield).
+Having such guessers makes attribute guessing more accurate and reliable than having generic guessing methods that doesn't  fit all topologies.
+
+Example uses of guessers
+------------------------
+
+Guessing using :ref:`guess_TopologyAttributes <guess_TopologyAttributes>` API
```

Comment:
Here as well -- although it might look nicer to just have plain `guess_TopologyAttributes` instead of the Sphinx markup!